Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-NOBELIUM_-_wide_scale_APT29_spear-phishing.yml

```diff
@@ -1599,6 +1486,292 @@ tasks:
     ignoreworker: false
     skipunavailable: false
     quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "35":
+    id: "35"
+    taskid: 780a4e7d-b965-4a47-8066-c35417f39940
+    type: playbook
+    task:
+      id: 780a4e7d-b965-4a47-8066-c35417f39940
+      version: -1
+      name: Block Indicators - Generic v3
+      description: |+
+        This playbook blocks malicious indicators using all integrations that are enabled, using the following sub-playbooks:
+
+        - Block URL - Generic v2
+        - Block Account - Generic v2
+        - Block IP - Generic v3
+        - Block File - Generic v2
+        - Block Email - Generic
+        - Block Domain - Generic
+
+      playbookName: Block Indicators - Generic v3
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "27"
+    scriptarguments:
+      AutoBlockIndicators:
+        complex:
+          root: inputs.AutoBlockIndicators
+      AutoCommit:
+        simple: "No"
+      CustomBlockRule:
+        simple: "True"
+      CustomURLCategory:
+        simple: XSOAR Remediation - Malicious URLs
+      DomainToBlock:
```

Comment:
need to remove `DomainToBlock`
Please ensure that the argument values on sub-playbook v3 match the argument values on the previous version of the sub-playbook Block Indicators - Generic v2.